src/java/test/org/apache/zookeeper/test/ZooKeeperTestClient.java

```diff
@@ -71,13 +71,14 @@ private void deleteZKDir(ZooKeeper zk, String nodeName)
     }
 
     List<String> children1 = zk.getChildren(nodeName, false);
-    List<String> c2 = zk.getChildren(nodeName, false, stat);
+    Stat stat2 = new Stat();
```

Comment:
Which rule has triggered this change?

Reply:
"SelfEquals" (see a few lines below in diff)

Reply:
Cool, thanks.